src/vaadin-message-input.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * @license
+ * Copyright (c) 2021 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+import { PolymerElement, html } from '@polymer/polymer/polymer-element.js';
+import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+import { ElementMixin } from '@vaadin/vaadin-element-mixin/vaadin-element-mixin.js';
+import '@vaadin/vaadin-button/src/vaadin-button.js';
+import '@vaadin/vaadin-text-field/src/vaadin-text-area.js';
+/**
+ * `<vaadin-message-input>` is a Web Component ...
+ */
+class MessageInputElement extends ElementMixin(ThemableMixin(PolymerElement)) {
+  static get template() {
+    return html`
+      <style>
+        :host {
+          align-items: flex-start;
+          box-sizing: border-box;
+          display: flex;
+          max-height: 50vh;
+          overflow: hidden;
+        }
+        vaadin-text-area {
+          align-self: stretch;
+          flex-grow: 1;
+          padding: 0;
+        }
+        vaadin-button {
+          flex-shrink: 0;
+          margin: 0;
+        }
+      </style>
+      <vaadin-text-area placeholder="Message"></vaadin-text-area>
+      <vaadin-button theme="primary contained">Send</vaadin-button>
```

Comment:
1. There should be a submit function that fires an event outside the component
2. This button should trigger that function.

Reply:
I'll abandon this comment as submit should be done in an own issue, #17.